src/Framework/MSBuildEventSource.cs

```diff
@@ -599,6 +599,30 @@ public void OutOfProcSdkResolverServiceRequestSdkPathFromMainNodeStop(int submis
             WriteEvent(80, submissionId, sdkName, solutionPath, projectPath, success, wasResultCached);
         }
 
+        [Event(81, Keywords = Keywords.All)]
```

Comment:
This will conflict with my instrumentation PR. Up the version preemptively? Also, can you add these to the doc?

Reply:
This PR will go in earlier, I believe, so I'd better bump it later in the instrumentation PR. We might want to add more events during the review process, so I would not know to which number I need to up the version now. I will add a comment to our MSBuild server PR, so we do not forget to bump it there. I will add it to the doc, thank you for mentioning, I was unaware of its existence.